Makefile

```diff
@@ -81,7 +81,7 @@ all: top-all
 YOSYS_SRC := $(dir $(firstword $(MAKEFILE_LIST)))
 VPATH := $(YOSYS_SRC)
 
-CXXFLAGS := $(CXXFLAGS) -Wall -Wextra -ggdb -I. -I"$(YOSYS_SRC)" -MD -D_YOSYS_ -fPIC -I$(PREFIX)/include
+CXXFLAGS := $(CXXFLAGS) -Wall -Wextra -Wimplicit-fallthrough=3 -ggdb -I. -I"$(YOSYS_SRC)" -MD -D_YOSYS_ -fPIC -I$(PREFIX)/include
```

Comment:
Not even this seems to work: https://travis-ci.org/github/YosysHQ/yosys/jobs/683030101#L850, even though it's clearly marked as fallthrough: https://github.com/YosysHQ/yosys/blob/584780d776c92bc91731dbc2710dd8d9a624dc70/backends/firrtl/firrtl.cc#L344-L347

Any ideas?